proxy/proxy.go

```diff
@@ -960,11 +950,10 @@ func (p *Proxy) makeBackendRequest(ctx *context, requestContext stdlibcontext.Co
 				status = http.StatusServiceUnavailable
 			}
 			p.tracing.setTag(ctx.proxySpan, HTTPStatusCodeTag, uint16(status))
-			return nil, &proxyError{err: err, code: status}
+			return nil, &proxyError{err: fmt.Errorf("net.Error during backend roundtrip to %s: timeout=%v temporary=%v: %w", req.URL.Host, nerr.Timeout(), nerr.Temporary(), err), code: status}
```

Comment:
maybe it would be useful to put the temporary between single quotes like: `temporary='%v'`, so that it wouldn't get mixed up with the main `err` value's string

Reply:
Temporary() returns bool so I think it's fine like this